main.js

```diff
@@ -12,4 +12,185 @@ closeMenu.addEventListener('click', () => {
 
 popUp.addEventListener('click', () => {
   popUp.classList.remove('pop-up');
-});
\ No newline at end of file
+});
+
+// cards
+const cardsUl = document.getElementById('cards');
+
+const projects = [
```

Comment:
- [ ]  Please I would like to remind you that you have four (4) different categories of projects. Therefore your popup must have four (4) different categories of projects. So, each Object in your Array with id:1, id:2, id:3, id:4 respectively contains its own information. 

- [ ]  Also, you need to see in your Popup only the one project (the one you selected). Not a Popup of scrolling projects. 

- From your PoPup

![s1](https://user-images.githubusercontent.com/63972670/162584208-6f2adcfa-46e0-4d63-821b-ca0d7a384656.JPG)